plugins/cros-ec/fu-cros-ec-firmware.h

```diff
@@ -36,6 +36,11 @@ typedef struct {
 	gchar				version[FU_FMAP_FIRMWARE_STRLEN];
 	gint32				rollback;
 	guint32				key_version;
+	guint64				image_idx;
 } FuCrosEcFirmwareSection;
 
+gboolean			 fu_cros_ec_firmware_pick_sections	(FuCrosEcFirmware *self,
+									 guint32 writeable_offset,
+									 GError **error);
+GArray				*fu_cros_ec_firmware_get_sections	(FuCrosEcFirmware *self);
```

Comment:
Is the whitespace here lined up?  (It looks off; but it might just be Github rendering it poorly - I didn't look in a local checkout).

Reply:
Hi @superm1 

It looks lined up to me in my editor, and when you look at it in non diff mode: 
https://github.com/fwupd/fwupd/blob/7527538bd74e0aa2ca1c6a51c3787bb8991e92bd/plugins/cros-ec/fu-cros-ec-firmware.h